tools/acncli/deployment/mdnc_azure-cni-manager.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: apps/v1
```

Comment:
why dont we update and resuse this https://github.com/Azure/azure-container-networking/blob/672cce9460be6e659c15356b6213ba8485271eb9/tools/acncli/deployment/manager_swift.yaml ?

Reply:
Because I don't know who else is consuming it... I don't want to cause side effects somewhere else.